### PR TITLE
chore(ReleaseInfo): update to v5.12.0

### PR DESCRIPTION
**Pull request by smbea**

### CODE REVIEW

**Comment by nikku** [comment]
I like the selection a lot. It goes without saying (and we link to the changelog) that there is loads of other things we worked on :).

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 1]
```diff
@@ -45,16 +45,24 @@ export function ReleaseInfo(props) {
...
-          The properties panel layout is now persisted across sessions.
-          This makes it easier to continue working where you left off.
+          <h4>Improvements to deployment tool</h4>
+          The deployment tool has received some UX improvements, such as: linking to a troubleshooting page with common connection problems; automatically infering port if not provided; and providing a deep link to the deployed process definition.
         </li>
         <li>
-          <h4>Message event templates support</h4>
-          This allows to use message catch event connectors in Camunda 8.
+          <h4>Unified bottom panel</h4>
+          The log and problems panels have been merged into a single resizeable bottom panel.
```
Comment (barmac) [inline client/src/plugins/version-info/ReleaseInfo.js:57]:
```suggestion
          The log and problems panels have been merged into a single resizable bottom panel.
```

**Comment by barmac** [comment]
LGTM

**Review by barmac — APPROVED** [review]

**Comment by nikku** [comment]
@barmac :arrow_right: :heavy_check_mark: :smile:

**Pull request merged**